s3/pkg/service/multipart.go

```diff
@@ -306,6 +327,40 @@ func (s *s3Service) CompleteMultipartUpload(ctx context.Context, in *pb.Complete
 		return err
 	}
 
+	md5Writer := md5.New()
+	var totalSize int64 = 0
+	for i := 0; i < len(in.CompleteParts); i++ {
+		if in.CompleteParts[i].PartNumber != int64(i+1) {
+			log.Infof("wrong order for part number. ")
```

Comment:
Is it better to use log.Errorf here?

Reply:
thanks, will use Errorln instead of infof